test.init.config

```diff
@@ -11,8 +11,8 @@
 			SecRule REQUEST_URI "test5" "id:2, deny, log, phase:1,status:403"
 			SecRule REQUEST_URI "test6" "id:4, deny, log, phase:3,status:403"
 		`
-		include test/sample1.conf
-		include test/sample2.conf
+		include testdata/sample1.conf
```

Comment:
nit: do we maybe want to avoid using the deprecated `include` directive and move it inside `directives`?

Reply:
I did the tiny change and moved them to `directives`